build.gradle

```diff
@@ -685,6 +685,7 @@ subprojects {
   }
 
   checkstyle {
+    configDirectory = rootProject.layout.projectDirectory.dir("checkstyle")
     configFile = new File(rootDir, "checkstyle/checkstyle.xml")
```

Comment:
we can perhaps change the file relative to configDirectory now instead of working relative to root directory.

Reply:
Good point. It turns out `${configDir}/checkstyle.xml` is [the default location](https://docs.gradle.org/current/dsl/org.gradle.api.plugins.quality.CheckstyleExtension.html#org.gradle.api.plugins.quality.CheckstyleExtension:configFile), so there is actually no need to specify `configFile` in this case. I pushed a commit to remove that line.